etc/modules.d/mpich.yaml

```diff
@@ -0,0 +1,24 @@
+name: mpich
+cli_arg: mpi
+help: Enable Cray MPICH Support
+env: ENABLE_MPICH_SS
+additional_args:
+  - -e SLURM_*
+  - -e PALS_*
+  - -e PMI_*
+  - -e LD_LIBRARY_PATH=/opt/udiImage/modules/mpich:/opt/udiImage/modules/mpich/dep
```

Comment:
just thinking about technical debt- should we make this a variable? when we retire shifter, maybe we'll want to use a different path?

Reply:
Yes.  I think we may be able to directly use the cray paths with podman.  We need to test that out.